### PR TITLE
Overlay.baseZIndex: fix ES module named export

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

File: js/ui/overlay.js [hunk 1]
```diff
@@ -1,2 +1,7 @@
 import Overlay from './overlay/ui.overlay';
-export default Overlay;
+
+const baseZIndex = Overlay.baseZIndex;
+export {
+    Overlay,
```
Comment (nightskylark) [inline js/ui/overlay.js:5]: Do we need this export? Overlay is private and all internal imports uses the `overlay/ui.overlay' module now. So who is it for?

File: js/ui/drawer/ui.drawer.rendering.strategy.overlap.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { animation } from './ui.drawer.animation';
 import DrawerStrategy from './ui.drawer.rendering.strategy';
 import $ from '../../core/renderer';
 import { move } from '../../animation/translator';
-import Overlay from '../overlay';
+import Overlay from '../overlay/ui.overlay';
```
Comment (vconst) [inline js/ui/drawer/ui.drawer.rendering.strategy.overlap.js]: Why is not `import { Overlay } from '../overlay'`? When we renovate Overlay only '../overlay' will be renovated in tests. The '../overlay/ui.overlay' import will return old jquery implementation.

**Review by Zedwag — DISMISSED** [review]

File: js/ui/overlay.js (hunk 1 above)
Comment (ksercs) [reply js/ui/overlay.js:5]: It was left for consistence with another editors to support flow with auto-replace to renovated component. But I forgot to use it in some places. fixed

File: js/ui/drawer/ui.drawer.rendering.strategy.overlap.js (hunk 2 above)
Comment (ksercs) [reply js/ui/drawer/ui.drawer.rendering.strategy.overlap.js]: It does not work in renovated wrappers. But I agree that we should use this approach in other cases

**Review by Zedwag — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**